utils/removeDiscordRoleFromUser.js

```diff
@@ -0,0 +1,44 @@
+const discordRolesModel = require("../models/discordactions");
+const discordServices = require("../services/discordService");
+const { logType } = require("../constants/logs");
+const { addLog } = require("../models/logs");
+
+/**
+ * Removes a Discord role from a user using Discord Id.
+ *
+ * @param {Object} userData - User data.
+ * @param {string} discordId - User's Discord ID.
+ * @param {string} roleid - Discord Role ID.
+ *
+ * @returns {Promise<Object>} - Result with success status and message.
+ */
+export const removeDiscordRoleFromUser = async (userData, discordId, roleid) => {
+  try {
+    const role = await discordRolesModel.isGroupRoleExists({ roleid });
```

Comment:
de-structure it on the fly

Reply:
Made the changes. Can you check it please?